app/models/cma_case.rb

```diff
@@ -0,0 +1,29 @@
+class CmaCase < Document
+  FORMAT_SPECIFIC_FIELDS = [
+    :opened_date,
+    :closed_date,
+    :case_type,
+    :case_state,
+    :market_sector,
+    :outcome_type
+  ]
+
+  attr_accessor *FORMAT_SPECIFIC_FIELDS
+
+  def initialize(params = {})
+    super(params, FORMAT_SPECIFIC_FIELDS)
+  end
+
+  def format
+    "cma_case"
+  end
+
+  def public_path
+    "/cma-cases"
+  end
+
+  def organisations
+    # Org name: Competition And Markets Authority
+    ["957eb4ec-089b-4f71-ba2a-dc69ac8919ea"]
```

Comment:
Good point, we would want the content-id. But looking at the current code and what's in content store, it doesn't seem that specialist docs send any links at all. Does this actually need to happen?